docs/api-keys.md

```
@@ -0,0 +1,141 @@
+Send the key in the `Authorization` header as a Bearer token:
+
+```bash
+curl -H "Authorization: Bearer swhk_aBcD1eFgH2iJkL3mNoP4qRsT5uVwX6yZ" \
+     https://example.com/wp-json/swh/v1/tickets
+```
+
+The format is `swhk_<32-character random string>`. The leading `swhk_` is intentional so the value is greppable in logs and secret scanners.
+
+Every response includes rate-limit headers (added in v4.1.0 Phase 7):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)